Index: src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\nspring.application.name=pomostudy\r\n\r\nspring.datasource.url=${DATABASE_URL}\r\nspring.datasource.username=${DATABASE_USERNAME}\r\nspring.datasource.password=${DATABASE_PASSWORD}\r\nserver.port=${PORT}\r\nspring.datasource.driver-class-name=org.postgresql.Driver\r\nspring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect\r\nspring.data.web.pageable.one-indexed-parameters=true\r\n\r\nspring.jpa.hibernate.ddl-auto=update\r\n\r\napi.security.token.secret=${JWT_SECRET:my-secret-key}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/src/main/resources/application.properties b/src/main/resources/application.properties
--- a/src/main/resources/application.properties	(revision 7cbdbf09106f977fde9623510ab60119dbb1d3bc)
+++ b/src/main/resources/application.properties	(date 1760309444889)
@@ -4,7 +4,7 @@
 spring.datasource.url=${DATABASE_URL}
 spring.datasource.username=${DATABASE_USERNAME}
 spring.datasource.password=${DATABASE_PASSWORD}
-server.port=${PORT}
+server.port=${PORT:8080}
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 spring.data.web.pageable.one-indexed-parameters=true
